src/bin/pgcopydb/pgsql.c

```diff
@@ -30,6 +30,14 @@
 #include "signals.h"
 #include "string_utils.h"
 
+/* Pipeline mode is only available in PostgreSQL 14 and later */
+#if PG_MAJORVERSION_NUM >= 14
+#define onPipelineMode(conn) (PQpipelineStatus(conn) == PQ_PIPELINE_ON)
+#else
+#warning Pipeline mode is only available in PostgreSQL 14 and later
+#define onPipelineMode(conn) (false)
+#endif
+
```

Comment:
I don't like the name `onPipelineMode` too much. What about `pqPipelineModeEnabled` or something like that?